src/app/shared/lowercase.ts

```diff
@@ -0,0 +1,25 @@
+import {Directive, Input, Output, EventEmitter, OnInit} from '@angular/core';
+
+@Directive({
+  selector: '[lowercase]',
+  host: {
+    '[value]': 'lowercase',
```

Comment:
Let's use HostListener & HostBinding rather than the host property of the decorator.

See: https://angular.io/guide/styleguide#hostlistenerhostbinding-decorators-versus-host-metadata